app/static/js/home/home.js

```diff
@@ -14,16 +14,18 @@ $(function() {
     });
 
 
-    var $image = $('.hero i');
     var md = new MobileDetect(window.navigator.userAgent);
-    var $devfestbanner = $('.devfest-banner');
-    var $nav = $('nav');
-    var $hero = $('.hero');
     if (md.mobile() == null) {
-        $(window).scroll(function(e){
+        var $image = $('.hero i');
+        var $devfestbanner = $('.devfest-banner');
+        var $nav = $('nav');
+        var $hero = $('.hero');
+
+        var scrollHandler = function() {
             var scrolled = $(window).scrollTop();
             console.log(scrolled);
```

Comment:
Should this line be in the public release? It spews out a stream of numbers in the console when users scroll.